libs/product/driver/magento/src/models/bundled-product.ts

```diff
@@ -23,10 +24,21 @@ export interface MagentoBundledProductItem {
 export interface MagentoBundledProductItemOption {
 	uid: string;
 	label: string;
-	price: number;
 	quantity: number;
-	can_change_quantity?: boolean;
 	is_default: boolean;
-	price_type?: MagentoPriceTypeEnum;
-	product?: MagentoSimpleProduct;
+	product: {
+		stock_status: MagentoProductStockStatusEnum;
+		price_range: {
+			maximum_price: {
+				regular_price: {
+					value: number;
+					currency: any;
+				};
+				discount: {
+					amount_off: number;
+					percent_off: number;
+				};
+			};
+		};
+	};
```

Comment:
I would make this a `MagentoBundledProductItemOptionProduct` model (terrible name) so that you can keep the transforms typed.

Reply:
added